boxes.lisp

```diff
@@ -355,11 +355,14 @@ N.B., Successive calls may return the same object."
             :type (integer 0)
             :accessor row-box-padding)
    (contents :initarg :contents
-             :accessor row-box-contents))
+             :accessor row-box-contents)
+   (align :initarg :align
+	  :accessor row-box-align))
   (:documentation "A horizontal concatenation of boxes."))
 
-(defun row-box (boxes &key (padding 0))
+(defun row-box (boxes &key (padding 0) (align 'baseline))
```

Comment:
I recommend using `:keywords` for these kinds of options. I would say `(align ':baseline)`. I would also define a new type somewhere for alignment options:

```
(deftype vertical-alignment ()
  `(member :baseline ...))
```

so we know explicitly possible alignment options. With this, you can specify the `:type` field in the slot definition.
 (See previous comment.)